acceptance/tests/rototillerTask/override_command_with_env.rb

```diff
@@ -0,0 +1,67 @@
+require 'beaker/hosts'
+require 'rakefile_tools'
+
+test_name 'C97827: can set envvar to override command name when using task.command' do
+  extend Beaker::Hosts
+  extend RakefileTools
+
+  step 'Test command with an override_env that has a value' do
+
+    override_env = 'BROCKSAMSON'
+    env_key = 'THIS_WAS_IN_ENV'
+    env_value = 'echo ' << env_key
+
+    @task_name    = 'command_flags_with_override'
+    rakefile_contents = <<-EOS
+$LOAD_PATH.unshift('/root/rototiller/lib')
+require 'rototiller'
+
+Rototiller::Task::RototillerTask.define_task :#{@task_name} do |t|
+    t.add_command({:name => 'echo', :override_env => '#{override_env}'})
+end
+    EOS
+    rakefile_path = create_rakefile_on(sut, rakefile_contents)
+    sut.add_env_var(override_env, env_value)
+
+    step 'Execute task defined in rake task' do
+      on(sut, "rake #{@task_name}", :accept_all_exit_codes => true) do |result|
+        # exit code & no error in output
+        assert(result.exit_code == 0, 'The expected exit code 0 was not observed')
+        assert_no_match(/error/i, result.output, 'An unexpected error was observed')
+
+        # command was used that was supplied by the override_env
+        assert_match(/^#{env_key}/, result.stdout, 'The correct command was not observed')
+      end
+    end
+  end
+
+  step 'Test command with an override_env that has a value' do
+
+    override_env = 'EMPTYENV'
+
+    @task_name    = 'command_flags_with_override'
+    rakefile_contents = <<-EOS
+$LOAD_PATH.unshift('/root/rototiller/lib')
+require 'rototiller'
+
+Rototiller::Task::RototillerTask.define_task :#{@task_name} do |t|
+    t.add_command do |c|
+      c.name = 'echo'
+      c.override_env = '#{override_env}'
+    end
+end
+    EOS
+    rakefile_path = create_rakefile_on(sut, rakefile_contents)
+
+    step 'Execute task defined in rake task' do
+      on(sut, "rake #{@task_name}", :accept_all_exit_codes => true) do |result|
+        # exit code & no error in output
+        assert(result.exit_code == 0, 'The expected exit code 0 was not observed')
+        assert_no_match(/error/i, result.output, 'An unexpected error was observed')
+
+        # flags should be echoed to command line
+        assert_match(/^--/, result.stdout, 'The correct command was not observed')
```

Comment:
does this work?

i don't see where it would output a line starting with `--`

```
          Expected /^--/ to match "\n\n\n".>
```

should probably echo something more pertinent to assert against